documentation/course/e2/README.md

```diff
@@ -732,45 +749,96 @@ TODO @jakubzehner: porównanie z NoSQL
     <th>Wady</th>
   </tr>
   <tr>
-    <th>Rozwiązanie 1</th>
+    <th>Bazy danych ACID</th>
+    <td>
+      <ul>
+        <li>Zapewnia synchronizację danych</li>
+        <li>Niezawodność</li>
+        <li>Bezpieczeństwo operacji</li>
+      </ul>
+    </td>
+    <td>
+      <ul>
+        <li>Skaluje się pionowo</li>
```

Comment:
**nit:** Seems weird to mention the ability to scale as a downside, how about "Nie skaluje się dobrze poziomo" instead of "skaluje się pionowo"? Or maybe "skaluje się głównie pionowo"?